software/communication.py

```diff
@@ -23,18 +24,21 @@ def __init__(self):
             
         print("Serial opened on port: ", self.port)
 
-    def send_inf(self, speed1, speedr, speed3, thrower_speed, disable_failsafe):
-        inf_out=struct.pack('<hhhHBH', speed1, speedr, speed3, thrower_speed, disable_failsafe, 0xAAAA)
-        self.Serial.write(inf_out)
-        #print("Data sent")
-        inf_in=self.Serial.read(8)
-        actual_speed1, actual_speedr, actual_speed3, feedback_delimiter = struct.unpack('<hhhH', inf_in)
-        #print("Real Data: " , actual_speed1, actual_speedr, actual_speed3, feedback_delimiter)
-        
+    def send_inf(self, speed1, speedr, speed3, thrower_speed, thrower_angle, grabber):

```

Comment:
I wonder if we can abstract the thrower_angle and grabber a little bit to a range of -1 to 1? Then it would be more meaningful to adjust them.